### PR TITLE
fixes banner listener minification bug, and fixes TOC links

### PR DESCRIPTION
**Pull request by scotthasbrouck**
@derrickstaten here ya go

@dmitrig01 lmk if this looks good

### CODE REVIEW

**Comment by derrickstaten** [comment]
looks good. I'll let @dmitrig01 give it a look since he knows better than I do. Minification is tough huh

File: src/3_branch.js [hunk 1]
```diff
@@ -1179,11 +1179,11 @@ if (WEB_BUILD) { // jshint undef:false
  * Remove the listener from observations, if it is present. Not that this function must be passed a referrence to the _same_ function that was passed to `branch.addListener()`, not just an identical clone of the function.
  *
  */
-/*** +TOC_ITEM #removelistener &.removeListener()& ^WEB ***/
+/*** +TOC_ITEM #removelistenerlistener &.removeListener()& ^WEB ***/
 	Branch.prototype['removeListener'] = function(listener) {
 		if (listener) {
 			this._listeners = this._listeners.filter(function(subscription) {
-				if (subscription.listener !== listener) { return subscription; }
+				if (subscription["listener"] !== listener) { return subscription; }
```
Comment (dmitrig01) [inline src/3_branch.js]: ack. we should use a closure compiler whatever-its-called for that

Comment (dmitrig01) [inline src/3_branch.js]: class or something

Comment (scotthasbrouck) [inline src/3_branch.js]: Oh right, I'll make that change and push it up tonight

**Comment by derrickstaten** [comment]
Is this finished now?

**Comment by scotthasbrouck** [comment]
No, sorry, I still need to make one more change. I'll do that next

File: src/3_branch.js (hunk 1 above)
Comment (scotthasbrouck) [inline src/3_branch.js]: @dmitrig01 I can't remember how to do this... it's not externs is it? Can you remember the name?

Comment (dmitrig01) [inline src/3_branch.js]: i think we put it in utils

**Comment by dmitrig01** [comment]
@scotthasbrouck check this out

**Comment by scotthasbrouck** [comment]
@dmitrig01 this is ready to go. Tests pass locally, and intermitinantly fail one one or two VMs on sauce

**Comment by dmitrig01** [comment]
:+1:

**Pull request merged**